data-es5.js

```diff
@@ -1170,6 +1225,8 @@ exports.tests = [
     return typeof Array.prototype.every == 'function';
   },
   res: {
+    es5shim: assign({ val: true }, sparseNote),
```

Comment:
This `assign` isn't really necessary - footnotes are keyed by ID, and using the same object will only create one footnote.